var/spack/repos/spack_repo/builtin/packages/py_torch/package.py

```diff
@@ -393,6 +405,14 @@ class PyTorch(PythonPackage, CudaPackage, ROCmPackage):
 
     # Fixes build error when ROCm is enabled for pytorch-1.5 release
     patch("rocm.patch", when="@1.5+rocm")
+    patch("PR152569-Update-spack-includes-2.5.patch", when="@2.5+rocm")
+    patch("PR152569-Update-spack-includes-2.6.patch", when="@2.6+rocm")
+    patch("PR152569-Update-spack-includes-2.7.patch", when="@2.7+rocm")
```

Comment:
Can you add a comment linking to this PR? If the PR has been merged, we may be able to directly use that PR patch instead of storing the file like you did below.